src/pages/messaging/agent-chat-api/v3.6/data-structures/index.mdx

```diff
@@ -335,6 +336,37 @@ System message event is native system event sent in specific situations.
 </Code>
 </Section>
 
+## System
+
+<Section>
+<Text>
+
+System event is an extension and a future replacement for [System message](#system-message). Similarly, it is a native system event sent in specific situations.
+
+### Request
+
+It is not possible to send System event in API version 3.6.
```

Comment:
```suggestion
It is not yet possible to send a System event in API version 3.6.
```

Reply:
@wfabjanczuk will it be possible in the future? I assume yes (as it would replace system-message which allows so), but if that's incorrect we'd have to remove 'yet'